recipes/viral_consensus/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# update environment variables
+export CFLAGS="$CFLAGS -I$PREFIX/include"
+export CPPFLAGS="$CPPFLAGS -I$PREFIX/include"
+export LDFLAGS="$LDFLAGS -L$PREFIX/lib"
+export CPATH=${PREFIX}/include
+export INCLUDES="-I${PREFIX}/include"
+export LIBPATH="-L${PREFIX}/lib"
+export LIBRARY_PATH=${PREFIX}/lib
+
+# compile ViralConsensus
+make CC=$CC CXX="${CXX}"
+if [ ! -d "$PREFIX/bin" ]; then
+    mkdir $PREFIX/bin;
+    export PATH=$PREFIX/bin:$PATH;
```

Comment:
No need to add to PATH for _build.sh_.
Most recipes just use `mkdir -p $PREFIX/bin` instead of a conditional.

Reply:
Thanks! I'll replace with just `mkdir -p` and remove the conditional